src/main/scala/intellij/haskell/psi/HaskellLayoutLexer.scala

```diff
@@ -36,17 +36,18 @@ class HaskellLayoutLexer(private val lexer: Lexer,
     override def toString = s"${elementType.toString} ($start, $end)"
 
     val isEOF: Boolean = elementType.isEmpty
-    val isCode: Boolean = elementType.exists(et => !nonCodeTokens.contains(et)) && !isEOF
+    def isCode: Boolean = elementType.exists(et => !nonCodeTokens.contains(et)) && !isEOF
 
-    val isNextLayoutLine: Boolean = isCode && line.columnWhereCodeStarts.contains(column)
+    def isNextLayoutLine: Boolean = isCode && line.columnWhereCodeStarts.contains(column)
```

Comment:
`isNextLayoutLine` can be also be `va`l if we decouple `Line` from `Token`. Much nicer.

Reply:
It is. I agree, but before spotting the other mistranslations, I tried to fix the lexer by modifying these codes and I forgot to change them back